### ISSUE TITLE
Add block form to model/edit.html

### ISSUE THREAD

**Issue by litnimax** [flask-admin/flask-admin#1006]
Imagine that I'd like to split the scrren in to col-md-6 divs and use left one for form editing and right one for my own staff. I could to this if there was  smth like this:
```
 {% block form %}
 {{ lib.render_form(form, return_url, extra(), form_opts) }}
{% endblock %}
```
In that case I coud move form to some other part.
It's so easy to add to the template:
```
{% block body %}
  {% block navlinks %}
  <ul class="nav nav-tabs">
    <li>
        <a href="{{ return_url }}">{{ _gettext('List') }}</a>
    </li>
    {%- if admin_view.can_create -%}
    <li>
        <a href="{{ get_url('.create_view', url=return_url) }}">{{ _gettext('Create') }}</a>
    </li>
    {%- endif -%}
    <li class="active">
        <a href="javascript:void(0)">{{ _gettext('Edit') }}</a>
    </li>
    {%- if admin_view.can_view_details -%}
    <li>
        <a href="{{ get_url('.details_view', id=model.id, url=return_url) }}">{{ _gettext('Details') }}</a>
    </li>
    {%- endif -%}
  </ul>
  {% endblock %}

  {{ lib.render_form(form, return_url, extra(), form_opts) }}
{% endblock %}
```

**Comment by pawl** [comment]
I don't think adding ```{% block form %}``` would hurt anything. Want to submit a pull request for it?

Don't forget to change it in both the bootstrap 2 and 3 templates.

**Comment by pawl** [comment]
Closed via #1088
